File: Anole/HTMLBodyElement.vb
Imports System
Imports System.Xml
Imports System.Windows.Forms
Imports System.Drawing
Namespace Anole
    ''' <summary>
    ''' Summary description for HTMLBodyElement.
    ''' </summary>
    Public Class HTMLBodyElement
        Inherits HTMLElement
        Public Sub New(ByRef document As HTMLDocument, parent As HTMLElement, xnode As XmlNode)
            Initialize(document, parent, xnode)
            ParseBackColor(xnode)
            document.BackColor = Me.BackColor
            Me.ForeColor = Me.ParseColor(xnode, "text", Color.Black)
            ParseChildren(xnode)
        End Sub
    End Class
End Namespace

File: Anole/HTMLBodyElement.cs
using System;
using System.Xml;
using System.Windows.Forms;
using System.Drawing;
namespace Anole
{
	/// <summary>
	/// Summary description for HTMLBodyElement.
	/// </summary>
	public class HTMLBodyElement : HTMLElement
	{
		public HTMLBodyElement(ref HTMLDocument document, HTMLElement parent, XmlNode xnode)
		{
			Initialize(ref document, parent, xnode);
			ParseBackColor(xnode);
			document.BackColor=this.BackColor;
			this.ForeColor=this.ParseColor(xnode,"text",Color.Black);
			ParseChildren(xnode);
		}
	}
}
